From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paolo <paolo.sabatino@gmail.com>
Date: Fri, 10 Mar 2023 19:15:10 +0100
Subject: [ARCHEOLOGY] sunxi, media: Realtek rtl8723cs driver consolidation
 (step 2) (#4924)

> X-Git-Archeology: > recovered message: > * sunxi: remove rtl8723cs patches from sunxi-edge (6.2)
> X-Git-Archeology: > recovered message: > * adjust minor pinephone patch
> X-Git-Archeology: > recovered message: > * media: remove rtl8723cs patches from media-edge
> X-Git-Archeology: > recovered message: > * media: remove rtl8723cs patches from media-current (6.1)
> X-Git-Archeology: > recovered message: > * sunxi: reinstate rtl8723cs patches for sunxi-6.1 and disable them
> X-Git-Archeology: > recovered message: > * sunxi: reinstate rtl8723cs patches for 6.2 and disable them
> X-Git-Archeology: - Revision fd022527f3780867236dbfbf62c6de4fa21d6886: https://github.com/armbian/build/commit/fd022527f3780867236dbfbf62c6de4fa21d6886
> X-Git-Archeology:   Date: Fri, 10 Mar 2023 19:15:10 +0100
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: sunxi, media: Realtek rtl8723cs driver consolidation (step 2) (#4924)
> X-Git-Archeology:
> X-Git-Archeology: - Revision e7f715091c9424dd48fb07a7e75a02af269fa26b: https://github.com/armbian/build/commit/e7f715091c9424dd48fb07a7e75a02af269fa26b
> X-Git-Archeology:   Date: Wed, 14 Jun 2023 05:36:26 +0200
> X-Git-Archeology:   From: Gunjan Gupta <viraniac@gmail.com>
> X-Git-Archeology:   Subject: allwinner: bump current to 6.1.33
> X-Git-Archeology:
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -496,6 +496,7 @@ &cpu3 {
 &csi {
 	pinctrl-0 = <&csi_pins>, <&csi_mclk_pin>;
 	status = "okay";
+	keep-power-in-suspend;
 
 	port {
 		#address-cells = <1>;
-- 
Armbian

